tradingstrategy/binance/downloader.py

```diff
@@ -369,12 +369,12 @@ def _fetch_lending_rates(
                 raise BinanceDataFetchError(
                     f"No data found for {asset_symbol} between {start_at} and {end_at}. Check your symbol matches with valid symbols in method description. \nResponse: {response.status_code} {response.text}"
                 )
-
+        
         dates = []
         interest_rates = []
         for data in response_data:
             dates.append(pd.to_datetime(data["timestamp"], unit="ms"))
-            interest_rates.append(float(data["dailyInterestRate"]))
+            interest_rates.append(float(data["dailyInterestRate"]) * 100 * 365) # convert daily to annual and multiply by 100
```

Comment:
Can you make 365 a labelled and commented constant, because different services use different length of a year (365 vs. 360)